스터디에 대해 궁금한 사항이 있어요 ~!

ttub-nii opened the pull request:
Github 스터디가 열린다는 소식을 보고 너무너무 좋은 스터디라는 생각이 들어 관심을 갖던 차에 몇 가지 궁금한 사항들이 생겨서 이렇게 질문을 남깁니다 ~~ 앞으로 스터디에 함께하지 못한다고 하더라도 넘 좋은 취지인 것 같아용 ~~ 스터디장님의 열정을 응원합니다~!~!

soogoon commented on QuA.md:
```diff
@@ -0,0 +1,17 @@
+### 질문 1 : 주 1회 스터디 모임은 오프라인으로만 진행되나요?
+온라인 스터디로 함께하고 싶은 참여자도 받으실 의향이 있는지 문의드립니다.  
+또는, 오프라인 스터디에 참여하지 않더라도 다른 스터디원들이 볼 수 있도록 자신이 공부한 내용을 문서로 정리해서 공유하고 같이 공유받는 방식도 괜찮은지 궁금합니다.
+
```
우선 어쩌다보니 아요친구들한테 먼저 스터디 모집을 뿌렸었는데 오늘 중으로 남은 인원을 채우기 위해 가장 최근기수인 25기 그룹에 스터디 모집 글을 올릴 것 같아요~
주 1회 오프라인으로만 진행됩니다! 다같이 모여서 공부하는 시너지를 위해서(혼자 공부안하는 승수...) 스터디를 열기로 마음 먹은 터라 온라인은 안합니다!
스터디 내용은 전부 md로 작성해서 여기에 올려놓을 것이기 때문에 누구나 봐도 괜찮습니다~! 이 스터디 포맷대로 따로 스터디를 열고싶어도 OK! 완전 자유에요 마음 껏 퍼가세요ㅎㅎ

soogoon commented on QuA.md:
```diff
@@ -0,0 +1,17 @@
+또는, 오프라인 스터디에 참여하지 않더라도 다른 스터디원들이 볼 수 있도록 자신이 공부한 내용을 문서로 정리해서 공유하고 같이 공유받는 방식도 괜찮은지 궁금합니다.
+
+### 질문 2 : 커리큘럼은 어떻게 정해지나요?
+정해진 커리큘럼에 따라 챕터를 담당한다면 커리큘럼은 스터디장이 정하는 것인지, 같이 정하는 것인지, 혹은 참고할만한 가이드라인이나 문서가 있는 것인지 궁금합니다.
```
스터디원이 정해지면 다같이 정할 거에요!
가이드라인 문서 아직 없습니다. 정말 다같이 머리 맡대고 지식공유 하는 식으로 해보려해용

soogoon commented on QuA.md:
```diff
@@ -0,0 +1,17 @@
+정해진 커리큘럼에 따라 챕터를 담당한다면 커리큘럼은 스터디장이 정하는 것인지, 같이 정하는 것인지, 혹은 참고할만한 가이드라인이나 문서가 있는 것인지 궁금합니다.
+
+### 질문 3 : 스터디에 불성실하게 임하는 스터디원에게 패널티가 부여되나요?
+매주 할당된 과제를 하지 않았거나, 과제를 했으나 그 내용이 빈약하거나, 오프라인으로 진행될 시에 참석하지 않는다면 주어지는 패널티가 있는지, 혹은 이 부분은 앞으로 스터디원이 모이고 나면 함께 상의할 부분인 것인지 궁금합니다.
```
과제는 딱히 생각해보지는 않았고 주마다 자기가 맡은 챕터를 나머지 스터디원에게 강의 해야하기 때문에 그 책임감으로 분명 준비 해오리라 믿어의심치 않습니다ㅎ
그렇다고 그 주차에서 준비해오는 인원이 아닌사람은 아무것도 하지않느냐! 이건 아니고 강의준비까지는 아니더라도 다같이 토론할 수 있게 다른 인원도 어느정도 공부를 해야겠죠?

You-hyeonyeong commented:
저도 할래욤

soogoon commented:
@You-hyeonyeong 오키~!

ttub-nii committed:
Fix curriculum

Through the first meeting, the presenters for each weeks and the topics to be presented were selected.

ttub-nii committed:
Create Repository-Settings-Github-Marketplace-Explore.md

ttub-nii committed:
Update README.md

ttub-nii committed:
Merge branch 'master' into master

ttub-nii committed:
update Marketplace 살펴보기

ttub-nii committed:
update Marketplace 살펴보기

ttub-nii committed:
Update Repository-Settings-Github-Marketplace-Explore.md

ttub-nii committed:
Merge branch 'master' of https://github.com/ttub-nii/Github-Cookbook

ttub-nii committed:
docs: Complete creation of the Marketplace & Explore

Finish writing rough copy of the First chapter *Marketplace n Explore

ttub-nii committed:
update 2nd Chapter

Repository Settings chapter is separated with Marketplace

ttub-nii committed:
Rename Repository-Settings-Github-Marketplace-Explore.md to Github-Marketplace-Explore.md

ttub-nii committed:
Create Repository-Settings.md

ttub-nii committed:
update Before Start

Add Preparation in advance list

ttub-nii committed:
update Features section

ttub-nii committed:
Set theme jekyll-theme-cayman

ttub-nii committed:
Set theme jekyll-theme-architect

ttub-nii committed:
Complete Options

Finish Writing articles & features about Options Menu

ttub-nii committed:
update Example using Apps

Codacy 적용 사례 및 후기 링크 추가

ttub-nii committed:
makeup All sessions include 'Branches'

브랜치 부분까지 내용 추가 & 그 전 챕터 모두 보강

ttub-nii committed:
Complete Repository Setting

Webhook 부터 Moderation 파트까지 추가 설명 작성 & 레이아웃 부분 수정
